app/models/reports/facility_state.rb

```diff
@@ -20,5 +20,9 @@ def self.region_id_field(region)
     def period
       @period ||= Period.month(month_date)
     end
+
+    scope :with_registered_assigned_or_follow_up_patients, -> {
```

Comment:
Consider `with_patients` or `with_active_patients`? ie, let's name the "what", not the "how".